### DISCUSSION TITLE
cannot configure my project with vercel.json

### DISCUSSION THREAD

**Discussion by ioeylim** [vercel/community#7063] [Help]
### Summary

Hi there, 

I've been trying to configure a cleaner routing on my website with the "rewrite" property, but the configuration is not applying on my deploy. I've made sure that vercel.json is located in the root directory and that I have followed the proper syntax. What could be the problem?

### Steps to Reproduce

<img width="534" alt="Screenshot 2024-06-02 at 1 58 24 PM" src="https://github.com/vercel/community/assets/113192435/55e51ee6-64d1-4264-a1af-6fb5715ec019">
<img width="397" alt="Screenshot 2024-06-02 at 1 58 56 PM" src="https://github.com/vercel/community/assets/113192435/2872e471-2485-428d-bd71-58f250c4147b">

**Comment by leerob** [comment]
Does it work if you add `.html` to the path? 

If so, you might need `cleanUrls`: https://vercel.com/docs/projects/project-configuration#cleanurls

**Reply by ioeylim** [reply]
Thanks for the reply! 

`cleanUrls` did help to take off the `.html ` in the routing, but still struggling to understand why `rewrites` is not properly applying. Any idea?
